### ISSUE TITLE
Verification Button should be focused

### ISSUE THREAD

**Issue by mylesdomingo** [calblueprint/alemany#43]
Verification screen should have a keyboard focused when entering the screen. 

Also, make the button pressable area for confirm larger.
